src/source-netmap.c

```diff
@@ -185,12 +170,21 @@ static SCMutex netmap_devlist_lock = SCMUTEX_INITIALIZER;
  */
 int NetmapGetRSSCount(const char *ifname)
```

Comment:
changes like this worry me a bit for backports, can we avoid these for the case where ppl did not specify to use the new code?